ctru-rs/src/test_runner.rs

```diff
@@ -15,8 +15,6 @@ use crate::services::Apt;
 /// runs all tests in series, "failing" on the first one to panic (really, the
 /// panic is just treated the same as any normal application panic).
 pub(crate) fn run(tests: &[&TestDescAndFn]) {
-    crate::init();
```

Comment:
we probably want to keep panic setup here, otherwise it will be hard to read output for any failing test

Reply:
The panic setup isn’t used for tests. The testing suite already implements similar code, mainly to read info about the test execution. You should know that, you wrote it! 😂

Reply:
I thought that we specifically called `init()` here to enforce the custom panic setup, particularly by the comment:

> (really, the panic is just treated the same as any normal application panic).

But I guess I'm remembering now that the output + panic get captured by the runner, then dumped when there's a failure (then loop and wait for user input). It has been a while since I wrote this stuff 😄

Reply:
I’m pretty sure we had an `init` there just for the `ps` stuff. Don’t worry, I was just teasing 😜